x-pack/qa/multi-project/xpack-rest-tests-with-multiple-projects/src/yamlRestTest/java/org/elasticsearch/multiproject/test/XpackWithMultipleProjectsClientYamlTestSuiteIT.java

```diff
@@ -26,6 +26,7 @@ public class XpackWithMultipleProjectsClientYamlTestSuiteIT extends MultipleProj
     public static ElasticsearchCluster cluster = ElasticsearchCluster.local()
         .distribution(DistributionType.DEFAULT)
         .name("yamlRestTest")
+        .module("test-multi-project")
```

Comment:
Nit: This is not necessary since the distribution is DEFAULT, i.e. all modules are included.

Reply:
do we pick DEFAULT here purposly? or would INTEG_TEST be enough?